### PR TITLE
Fix(docs) broken links

### PR DESCRIPTION
**Pull request by dondevo**
Link broken on gatsbyjs.org but working in relative repo.

### CODE REVIEW

File: docs/contributing/setting-up-your-local-dev-environment.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ Yarn is a package manager for your code, similar to [NPM](https://www.npmjs.com/
 - Set up repo and install dependencies: `yarn run bootstrap`
 - Make sure tests are passing for you: `yarn test`
 - Create a topic branch: `git checkout -b topics/new-feature-name`
-- See [docs setup instructions](/contributing/docs-contributions#docs-site-setup-instructions) below for docs-only changes.
+- See [docs setup instructions](https://github.com/gatsbyjs/gatsby/blob/master/docs/contributing/docs-contributions.md#docs-site-setup-instructions) below for docs-only changes.
```
Comment (pieh) [inline docs/contributing/setting-up-your-local-dev-environment.md]: This seems to be working for me without the change

File: docs/contributing/setting-up-your-local-dev-environment.md [hunk 2]
```diff
@@ -35,7 +35,7 @@ Yarn is a package manager for your code, similar to [NPM](https://www.npmjs.com/
 - For each of your Gatsby test sites, run the `gatsby-dev` command inside the test site's directory to copy
   the built files from your cloned copy of Gatsby. It'll watch for your changes
   to Gatsby packages and copy them into the site. For more detailed instructions
-  see the [gatsby-dev-cli README](/packages/gatsby-dev-cli/) and check out the [gatsby-dev-cli demo video](https://www.youtube.com/watch?v=D0SwX1MSuas).
+  see the [gatsby-dev-cli README](https://github.com/gatsbyjs/gatsby/tree/master/packages/gatsby-dev-cli) and check out the [gatsby-dev-cli demo video](https://www.youtube.com/watch?v=D0SwX1MSuas).
```
Comment (pieh) [inline docs/contributing/setting-up-your-local-dev-environment.md]: maybe better link to https://www.npmjs.com/package/gatsby-dev-cli ?

File: docs/contributing/setting-up-your-local-dev-environment.md (hunk 1 above)
Comment (DSchau) [reply docs/contributing/setting-up-your-local-dev-environment.md]: Yeah - let's keep this as is. Remember that our documentation is intended to be viewed on gatsbyjs.org, not in Github!

```suggestion
- See [docs setup instructions](/contributing/docs-contributions.md#docs-site-setup-instructions) below for docs-only changes.
```

File: docs/contributing/setting-up-your-local-dev-environment.md (hunk 2 above)
Comment (DSchau) [reply docs/contributing/setting-up-your-local-dev-environment.md]:
```suggestion
  see the [gatsby-dev-cli README](https://www.npmjs.com/package/gatsby-dev-cli) and check out the [gatsby-dev-cli demo video](https://www.youtube.com/watch?v=D0SwX1MSuas).
```

Comment (dondevo) [reply docs/contributing/setting-up-your-local-dev-environment.md]: Updated, thanks.

**Pull request merged**